audit/src/main/scala/tech/beshu/ror/audit/instances/DefaultAuditLogSerializerV1.scala

```diff
@@ -17,82 +17,42 @@
-      Some(createEntry(matched = true, "FORBIDDEN", reason, responseContext.duration, requestContext, None))
-    case Forbidden(requestContext) =>
-      Some(createEntry(matched = false, "FORBIDDEN", "default", responseContext.duration, requestContext, None))
-    case RequestedIndexNotExist(requestContext) =>
-      Some(createEntry(matched = false, "INDEX NOT EXIST", "Requested index doesn't exist", responseContext.duration, requestContext, None))
-    case Errored(requestContext, cause) =>
-      Some(createEntry(matched = false, "ERRORED", "error", responseContext.duration, requestContext, Some(cause)))
-  }
-
-  private def createEntry(matched: Boolean,
-                          finalState: String,
-                          reason: String,
-                          duration: FiniteDuration,
-                          requestContext: AuditRequestContext,
-                          error: Option[Throwable]) = {
-    new JSONObject()
-      .put("match", matched)
-      .put("block", reason)
-      .put("id", requestContext.id)
-      .put("final_state", finalState)
-      .put("@timestamp", timestampFormatter.format(requestContext.timestamp))
-      .put("correlation_id", requestContext.correlationId)
-      .put("processingMillis", duration.toMillis)
-      .put("error_type", error.map(_.getClass.getSimpleName).orNull)
-      .put("error_message", error.map(_.getMessage).orNull)
-      .put("content_len", requestContext.contentLength)
-      .put("content_len_kb", requestContext.contentLength / 1024)
-      .put("type", requestContext.`type`)
-      .put("origin", requestContext.remoteAddress)
-      .put("destination", requestContext.localAddress)
-      .put("xff", requestContext.requestHeaders.getValue("X-Forwarded-For").flatMap(_.headOption).orNull)
-      .put("task_id", requestContext.taskId)
-      .put("req_method", requestContext.httpMethod)
-      .put("headers", requestContext.requestHeaders.names.asJava)
-      .put("path", requestContext.uriPath)
-      .put("user", SerializeUser.serialize(requestContext).orNull)
-      .put("impersonated_by", requestContext.impersonatedByUserName.orNull)
-      .put("action", requestContext.action)
-      .put("indices", if (requestContext.involvesIndices) requestContext.indices.toList.asJava else List.empty.asJava)
-      .put("acl_history", requestContext.history)
-      .mergeWith(requestContext.generalAuditEvents)
-  }
-
-  private implicit class JsonObjectOps(val mainJson: JSONObject) {
-    def mergeWith(secondaryJson: JSONObject): JSONObject = {
-      jsonKeys(secondaryJson).foldLeft(mainJson) {
-        case (json, name) if !json.has(name) =>
-          json.put(name, secondaryJson.get(name))
-        case (json, _) =>
-          json
-      }
-    }
+}
 
-    private def jsonKeys(json: JSONObject) = {
-      Option(JSONObject.getNames(json)).toList.flatten
-    }
-  }
+object DefaultAuditLogSerializerV1 {
+  val defaultV1AuditFields: Map[String, AuditFieldValue] = Map(
```

Comment:
Each serializer now uses the `BaseAuditLogSerializer.serialize`, with various sets of arguments:
- fields (serializers have the predefined list of fields, like this one; only `Configurable` serializer gets its list of fields from config
- `AllowedEventSerializationMode` (that was the missing functionality - to log all ALLOWED events, even those for rules with INFO level verbose; there are new serializers `ReportingAllEvents` with and without query; for `Configurable` serializer this setting is defined in config)